drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -21,23 +21,42 @@
 LOG_MODULE_REGISTER(bq274xx, CONFIG_SENSOR_LOG_LEVEL);
 
 /* subclass 64 & 82 needs 5ms delay */
-#define BQ274XX_SUBCLASS_DELAY 5
+#define BQ274XX_SUBCLASS_DELAY_MS 5
```

Comment:
```suggestion
#define BQ274XX_SUBCLASS_DELAY K_MSEC(5)
```
This will require other code changes (`k_msleep` -> `k_sleep`) but using this strategy the units are effectively built into the define value.

Reply:
See also https://github.com/zephyrproject-rtos/zephyr/pull/60117/files#r1257606462

Reply:
If you like this approach you could also apply to `PIN_DELAY_TIME` and `INIT_TIME`

Reply:
Yeah I prefer k_sleep as well, the driver delay strategy has to be reworked though. Anyway, why not, I've switched this to `K_MSEC` and added one for `50`.

Reply:
Is it worth also addressing `PIN_DELAY_TIME` and `INIT_TIME` while we're here?

Reply:
Honestly I have no idea what are those two references. Is that referring to the 1100ms delay after config update and some other delay I have yet to bump into? How about  we wrap this PR at this and then you followup with more changes on your side? If you know how that should be done already it may be faster than iterating through comments.

Reply:
Sorry I see that now, not sure how I missed the other two defines, yes I'll change them.

Reply:
All done.